services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/mounted_fs.py

```diff
@@ -86,22 +86,22 @@ def disk_inputs_path(self) -> Path:
     def disk_outputs_path(self) -> Path:
         return _ensure_path(self._dy_volumes / self.outputs_path.relative_to("/"))
 
-    def disk_state_paths(self) -> Iterator[Path]:
+    def disk_state_paths_iter(self) -> Iterator[Path]:
```

Comment:
Q: do we **really** need iterators for a 2, 3 paths here? Note that iterators are handy when a lot of items but they also can be error prone since you "consume" them once you read them.

Reply:
I will not change it now. Just renamed for clarity. In a future refactor I will do this.